src/components/user/EditProfile/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSession, useDispatch } from 'hooks';
+import ProfileContainer from 'assets/profile-container.png';
+import Profile from 'assets/profile-icon.png';
+import { editProfile } from 'state/actions/userActions';
+import EditProfileForm from './EditProfileForm';
+
+const EditProfile = () => {
+  const { user } = useSession();
+  const editProfileRequest = useDispatch(editProfile);
+
+  return (
+    <div className="edit-profile-container">
+      <div className="profile-container">
+        <div className="profile-icon-container">
+          <img className="profile-icon-bg" src={ProfileContainer} alt="Profile Container" />
+          <img className="profile-icon" src={Profile} alt="Profile Icon" />
+        </div>
+        <p className="username">{user.email}</p>
+      </div>
+
+      <EditProfileForm onSubmit={editProfileRequest} user={user} />
+
+      <div>
+        <p>Delete my TARGET account</p>
```

Comment:
Can we use I18n here? Maybe in the `img alt` attribute too

Reply:
Yes! i18n would be added in next PR as it involves the delete user account functionality as a whole